Makefile

```diff
@@ -187,19 +187,30 @@ ui-release:
 ui-zip-and-release: ui-zip-contents ui-release ## Zip and release prod static ui site
 
 
-##
-release:
-	@python release.py --operation check
+release-api:
+	@python release.py --operation check --type api
 	@git checkout ${commit}
 	@git tag -a "${version}" -m "Release tag for version ${version}"
 	@git checkout -
 	@git push origin ${version}
-	@python release.py --operation create-changelog
-	@gh release create ${version} -F latest_release_changelog.md
+	@python release.py --operation create-changelog check --type api
+	@gh release create ${version} -F latest_release_changelog_api.md -t "API Release"
 	@rm -rf latest_release_changelog.md
 
+release-sdk:
+	@python release.py --operation check --type sdk
```

Comment:
Can we turn this into another command or a bash function/file to reduce the duplication in the code and instead pass everything through as command line arguments